src/Services/PaymentOrder/models/PaymentOrderResponse.d.ts

```diff
@@ -0,0 +1,14 @@
+import { PaymentOrder } from "../PaymentOrder";
+import { Operation } from "../../../models/Operation";
+import { Payment } from "../../CardPayment/models/Payment";
+
+export class PaymentOrderResponse {
```

Comment:
```suggestion
export interface PaymentOrderResponse {
```

Is this an interface?